pennylane/queuing.py

```diff
@@ -246,7 +250,130 @@ def _get_info(self, obj):
 
         return self._queue[obj]
 
+    append = _append
+    remove = _remove
+    update_info = _update_info
+    get_info = _get_info
```

Comment:
```suggestion
    # Overwrite the inherited class methods, so that if annotated_queue.append is called,
    # it is appended to the instantiated queue (rather than being added to the
    # currently active queuing context, which may be a different queue).
    append = _append
    remove = _remove
    update_info = _update_info
    get_info = _get_info
```